docs/user-guide/mvd-configuration.md

```diff
@@ -675,37 +737,43 @@ When you use the default Zowe SMP/E or convenience build configuration, you do
 To configure Zowe for MFA with a configuration other than the default, take the following steps:
 
 1. Choose an App Server security plugin that is compatible with MFA. The [apiml-auth, zss-auth, and zosmf-auth](#session-duration-and-expiration) plugins are all compatible.
-2. Locate the App Server's configuration file in `$INSTANCE_DIR/workspace/app-server/serverConfig/server.json`
-3. Edit the configuration file to modify the section `dataserviceAuthentication`.
-
-4. Set `defaultAuthentication` to the same category as the plugin of choice, for example:
-    * **apiml-auth**: "apiml"
+2. Open the instance configuration file, either instance.env or zowe.yaml.
+3. Set environment variable `ZWED_dataserviceAuthentication_defaultAuthentication` to the same category as the plugin of choice, for example:
```

Comment:
I think it is components.app-server.dataserviceAuthentication.defaultAuthentication in zowe.yaml